packages/core/test/unit/child-proxy/id-generator.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { expect } from 'chai';
+
+import { IdGenerator } from '../../../src/child-proxy/id-generator.js';
+
+describe(IdGenerator.name, () => {
+  let idGenerator: IdGenerator;
+  beforeEach(() => {
+    idGenerator = new IdGenerator();
+  });
+  it('should increment workerId on calling `next`', async () => {
```

Comment:
Since we control the SUT (System Under Test, namely the `IdGenerator` class), we can test that the first call to `next` provides the value `0`.